stdlib/REPL/src/REPL.jl

```diff
@@ -1704,21 +1704,11 @@ function ends_with_semicolon(code)
     return semi
 end
 
-"""
-    banner(io::IO = stdout, preferred::Symbol = :full)
-
-Print the "Julia" informative banner to `io`, using the `preferred` variant
-if reasonable and known.
-
-!!! warning
-    The particular banner selected by `preferred` is liable to being changed
-    without warning. The current variants are: `:tiny`, `:short`, `:narrow`, and `:full`.
-"""
-function banner(io::IO = stdout, preferred::Symbol = :full)
-    commit_string = if Base.GIT_VERSION_INFO.tagged_commit
-        Base.AnnotatedString(TAGGED_RELEASE_BANNER, :face => :shadow)
```

Comment:
The specific failure that was getting swallowed and turned into a deadlock was this (courtesy of @xal-0):
```
Unhandled Task ERROR: UndefVarError: `TAGGED_RELEASE_BANNER` not defined in `REPL`
Suggestion: check for spelling errors or missing imports.
Stacktrace:
 [1] banner(io::IO, preferred::Symbol)
   @ REPL ~/c/julia-1.13/usr/share/julia/stdlib/v1.13/REPL/src/REPL.jl:1719
 [2] run_std_repl(REPL::Module, quiet::Bool, banner::Symbol, history_file::Bool)
   @ Base ./client.jl:487
 [3] (::REPL.Precompile.var"#repl_workload##11#repl_workload##12"{Base.TTY, Base.TTY, Base.TTY})()
   @ REPL.Precompile ~/c/julia-1.13/usr/share/julia/stdlib/v1.13/REPL/src/precompile.jl:144
```
So this code path requires a tag in order to be exercised and thus observe the failure.

Reply:
Sigh, how sneaky...